modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -221,371 +162,42 @@ class EditorCommands {
    * @return {Boolean} true/false if the command is supported or not.
    */
   public queryCommandSupported(command: string): boolean {
-    command = command.toLowerCase();
-
-    if (this.commands.exec[command]) {
-      return true;
+    if (this.editor.quirks.isHidden() || this.editor.removed) {
+      return false;
     }
 
-    // Browser commands
-    try {
-      return this.editor.getDoc().queryCommandSupported(command);
-    } catch (ex) {
-      // Fails sometimes see bug: 1896577
+    const lowerCaseCommand = command.toLowerCase();
+    if (this.commands.exec[lowerCaseCommand]) {
+      return true;
     }
 
     return false;
   }
 
+  /**
+   * Adds a custom query state command to the editor, you can also override existing commands with this method.
+   * The command that you add can be executed with queryCommandState function.
```

Comment:
```suggestion
   * Adds a custom query state command to the editor. This function can also be used to override existing commands.
   * The added command can be executed with the <code>queryCommandState<code> function.
```
`You` and other pronouns is something that should almost always be avoided in docs: https://www.tiny.cloud/docs/configure/contributing-docs/#pronouns